pages/linux/feedreader.md

```diff
@@ -1,7 +1,7 @@
 # feedreader
 
 > A GUI desktop RSS client.
-> More information: <https://jangernert.github.io/FeedReader/>.
+> More information: <https://github.com/jangernert/FeedReader>.
```

Comment:
It will be good to note that this command is no longer maintained.
```suggestion
> Note: FeedReader is no longer being maintained.
> More information: <https://github.com/jangernert/FeedReader>.
```
or also note that it was succeeded by NewsFlash
```suggestion
> Note: FeedReader is no longer being maintained. NewsFlash has succeeded it.
> More information: <https://github.com/jangernert/FeedReader>.
```